packages/wix-storybook-utils/stories/story-with-sections.story.tsx

```diff
@@ -24,6 +25,28 @@ export default {
   component: Component,
   componentPath: './component.js',
   sections: [
+    tab({
+      title: 'Something something',
+      sections: [
+        columns({
+          title: 'Septyni astuoni keturiolika',
+          items: [
+            description({ text: `🔨 To trigger an operation.` }),
+            importExample({
+              source: "import Button from 'wix-style-react/Button';",
+            }),
+          ],
+        }),
+        table({
+          title: 'Included Components',
+          rows: [
+            ['&lt;FormField/&gt;', 'Layout component for form elements'],
```

Comment:
you shouldn't need to escape, did it not work?
```suggestion
            ['<FormField/>', 'Layout component for form elements'],
```